wmfdata/mariadb.py

```diff
@@ -24,21 +26,21 @@ def connect(db, use_x1=False):
     # database is located on x1.
     if db == "wikishared":
         use_x1 = True
-    
+
```

Comment:
I blame my editor, and I am sorry.

Reply:
I also got undefined variable problems without the `connection=None` line.

Reply:
> I blame my editor, and I am sorry.

In that case, I'll withdraw the complaint I filed with the code police 😊

> I also got undefined variable problems without the connection=None line.

As I said, I think the cleanup is actually good since it makes it follow PEP-8. Same with this; I get those problems too and I think it's a good fix! Just in a separate pull request as a kindness to my limited lizard brain.